CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.6)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
We need to use CMake v3.1+ in testing for our coverage submission to work.

Since it's not feasible to make that our minimum requirement since it's not in any of the package managers, we should use the oldest possible CMake version that will successfully configure and build girder, which I believe is 2.8.6, but we can build 2.8.6 and see if that's true.